shuffle/cli/src/console.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::shared;
+use anyhow::Result;
+use std::{path::Path, process::Command};
+
+pub fn handle(project_path: &Path) -> Result<()> {
+    let _config = shared::read_config(project_path)?;
+
+    // TODO: Fix hardcoding of mod.ts by copying it into project path and referencing
+    // it via project_path.join(...). Remove Message pkg hardcode and iterate over pkgs.
```

Comment:
Update comment to include TODO to pass in project name

Also probably best to rename Message to Project

Reply:
Removed TODO and renamed package to `main/` to mirror rust.